controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -469,6 +483,14 @@ def test_delete_snapshot_call_rmfcmap(self):
 
         self.svc.client.svctask.rmfcmap.assert_called_once_with(object_id="test_fc_id", force=True)
 
+    def test_delete_snapshot_in_hyperswap(self):
```

Comment:
it's not completely clear from the tests name which feature should or should not work with hyperswap.
if the tests purpose is only to ensure that object (volume/snapshot) deletion doesn't delete hyperswap fcmaps, then it should be reflected in the tests name, e.g. `test_delete_snapshot_does_not_delete_hyperswap_fcmaps`

Reply:
Done

Reply:
thanks.
specifically for expand, since it also does `expandvdisksize.assert_called_once_with`, the previous name was ok

Reply:
Done